### PR TITLE
Update README.md and Add README_zh-CN.md

### PR DESCRIPTION
**Pull request by ToumaKazusa3**

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+[English](/README.md) | 简体中文
+
+文档: https://mmtracking.readthedocs.io/
+
+## 简介
+
+MMTracking是一款基于 PyTorch 的视频目标感知开源工具箱，是 [OpenMMLab](http://openmmlab.org/) 项目的一部分。
+
+主分支代码目前支持 **PyTorch 1.3 到 1.7** 版本。
```
Comment (OceanPang) [inline README_zh-CN.md:19]: Change the version to `PyTorch 1.3以上的版本`, also revise the English version.

File: README_zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+[![codecov](https://codecov.io/gh/open-mmlab/mmtracking/branch/master/graph/badge.svg)](https://codecov.io/gh/open-mmlab/mmtracking)
+[![license](https://img.shields.io/github/license/open-mmlab/mmtracking.svg)](https://github.com/open-mmlab/mmtracking/blob/master/LICENSE)
+
+[English](/README.md) | 简体中文
+
+文档: https://mmtracking.readthedocs.io/
+
+## 简介
+
+MMTracking是一款基于 PyTorch 的视频目标感知开源工具箱，是 [OpenMMLab](http://openmmlab.org/) 项目的一部分。
```
Comment (OceanPang) [inline README_zh-CN.md:17]: Remove the unnecessary space, like the two next to `PyTorch`. The same to the entire doc.

File: README_zh-CN.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  **简洁**：MMTracking 与其他 OpenMMLab 平台充分交互。MMTracking 建立在 [MMDetection](https://github.com/open-mmlab/mmdetection)
+  的基础上，我们只需要修改配置文件就可以使用任何检测器。
+
+  **高效**：MMTracking 所有操作都在GPU上运行。训练和推理速度比其他实现更高效或相持平。
+
+  **强大**：MMTracking 复现了SOTA性能的模型，部分实现精度超出官方版本。
+
+## 许可
+
+该项目开源自 [Apache 2.0 license](/LICENSE)
```
Comment (OceanPang) [inline README_zh-CN.md:46]: "该项目遵循***开源协议。"

File: README_zh-CN.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+  MMTracking 是首个开源一体化视频目标感知工具箱，同时支持视频目标检测，单目标跟踪和多目标跟踪等多种任务和算法。
+
+- **模块化设计**
+
+  MMTracking 将统一的视频目标感知框架解耦成不同的模块组件，通过组合不同的模块组件，用户可以便捷地构建自定义的视频目标感知模型。
+
+- **简洁，高效，强大**
+
+  **简洁**：MMTracking 与其他 OpenMMLab 平台充分交互。MMTracking 建立在 [MMDetection](https://github.com/open-mmlab/mmdetection)
```
Comment (OceanPang) [inline README_zh-CN.md:37]: “建立在” -> “充分复用了MMDetection中的已有模块”

File: README_zh-CN.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+  的基础上，我们只需要修改配置文件就可以使用任何检测器。
+
+  **高效**：MMTracking 所有操作都在GPU上运行。训练和推理速度比其他实现更高效或相持平。
+
+  **强大**：MMTracking 复现了SOTA性能的模型，部分实现精度超出官方版本。
```
Comment (OceanPang) [inline README_zh-CN.md:42]: 可以参考知乎文档将这部分polish的更加充实。

File: README_zh-CN.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+该项目开源自 [Apache 2.0 license](/LICENSE)
+
+## 更新记录
+
+v0.5.1 版本已于 2021 年 2 月 1 日发布，可通过查阅 [更新日志](/docs/changelog.md) 了解更多细节以及发布历史。
+
+## 基准测试与模型库
+
+各个模型的结果和设置都可以在 [模型库](docs/model_zoo.md) 页面中查看。
```
Comment (OceanPang) [inline README_zh-CN.md:54]: 前面增加一个“本工具箱支持的”

File: README_zh-CN.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+- [x] [SiameseRPN++](configs/sot/siamese_rpn) (CVPR 2019)
+
+## 安装
+
+请参考 [安装指南](docs/install.md) 进行安装
+
+## 教程
+
+请参考 [数据集](docs/dataset.md) 和 [快速开始](docs/quick_run.md) 了解 MMAction2 的基本使用。MMTracking 也提供了更详细的 [教程](docs/tutorials/)
```
Comment (OceanPang) [inline README_zh-CN.md:77]: 这里还是MMAction2

File: README_zh-CN.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+- [MMCV](https://github.com/open-mmlab/mmcv): OpenMMLab 计算机视觉基础库
+- [MMClassification](https://github.com/open-mmlab/mmclassification): OpenMMLab 图像分类工具箱与测试基准
+- [MMDetection](https://github.com/open-mmlab/mmdetection): OpenMMLab 检测工具箱与测试基准
+- [MMDetection3D](https://github.com/open-mmlab/mmdetection3d): OpenMMLab 新一代通用3D目标检测平台
+- [MMSegmentation](https://github.com/open-mmlab/mmsegmentation): OpenMMLab 语义分割工具箱与测试基准
+- [MMAction2](https://github.com/open-mmlab/mmaction2): OpenMMLab 新一代视频理解工具箱与测试基准
+- [MMTracking](https://github.com/open-mmlab/mmtracking): OpenMMLab 一体化视频目标感知平台
+- [MMPose](https://github.com/open-mmlab/mmpose): OpenMMLab 姿态估计工具箱与测试基准
+- [MMEditing](https://github.com/open-mmlab/mmediting): OpenMMLab 图像视频编辑工具箱
+- [MMOCR](https://github.com/open-mmlab/mmocr): OpenMMLab 全流程文字检测识别理解工具包.
```
Comment (OceanPang) [inline README_zh-CN.md:111]: 英文版本也加一下MMOCR吧

**Review by OceanPang — APPROVED** [review]

**Comment by OpenMMLab-Coodinator** [comment]
Hi! @ToumaKazusa3 First of all, we want to express our gratitude for your significant PR in this project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

If you are Chinese or have WeChat，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**